packages/neuron-ui/src/services/localCache.ts

```diff
@@ -128,3 +129,25 @@ export const syncRebuildNotification = {
     return window.localStorage.getItem(LocalCacheKey.SyncRebuildNotification)
   },
 }
+
+export enum Theme {
+  Light = 'light',
+  Dark = 'dark',
+}
+
+export const theme = {
+  save: (type: Theme) => {
+    window.localStorage.setItem(LocalCacheKey.Theme, type)
```

Comment:
We should read theme preference from `prefers-color-scheme` and `nativeTheme.shouldUseDarkColors` and set theme with `nativeTheme.themeSource`

Ref: 
1. https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
2. https://www.electronjs.org/zh/docs/latest/api/native-theme#nativethemethemesource